ethereum/scripts/fetchDeployedAddrs.js

```diff
@@ -0,0 +1,25 @@
+// Example usage:
+// truffle exec fetchDeployedAddrs.js [user-address]
+// truffle exec fetchDeployedAddrs.js --network ropsten
+
+const ETHApp = artifacts.require("ETHApp")
+const ERC20App = artifacts.require("ERC20App")
+const TestToken = artifacts.require("TestToken")
+
+module.exports = async () => {
+    try {
+        const eth20AppInstance = await ETHApp.deployed()
+        const erc20AppInstance = await ERC20App.deployed()
+        const tokenInstance = await TestToken.deployed();
+
+        const jsonStr = "{" +
```

Comment:
It would be cleaner to serialize an object using `console.log(JSON.stringify(obj, null, 2))`, where `obj = {  ethApp: ... }`

Reply:
Updated in https://github.com/Snowfork/polkadot-ethereum/pull/112/commits/659629678494987bc0f9c00e341cac161a3341b0